src/chart/hawkular-chart-directive.ts

```diff
@@ -446,11 +447,11 @@ module Charts {
           function formatBucketedChartOutput(response) {
             //  The schema is different for bucketed output
             if (response) {
-
               return response.map((point:IChartDataPoint) => {
+                var timestamp = point.start + (point.start - point.end)/2;
```

Comment:
Ah, good to know. This is one of those things that changed a while back and another reason why the standalone charts weren't working. Thanks!

LGTM.